service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -90,10 +92,18 @@
     @MockBean
     private InboxLookupService inboxLookupService;
 
+    @MockBean
+    private Time time;
+
     DraftOrdersControllerTest() {
         super("draft-standard-directions");
     }
 
+    @BeforeEach
+    public void init() {
+        when(time.now()).thenReturn(LocalDateTime.now());
+    }
+
```

Comment:
This shouldn't be needed, we already have a fixed time configuration that is used for testing

Reply:
`service/src/test/java/uk/gov/hmcts/reform/fpl/utils/FixedTimeConfiguration.java`

Reply:
Or is that not pulled in for the integration tests?

Reply:
you are right, fixed time is already used, done